src/components/keymap/header/keymap-header.component.scss

```diff
@@ -7,7 +7,7 @@
             cursor: pointer;
 
             &:hover {
-                color: #b9b6b6;
+                color: $icon-hover;
```

Comment:
I know, but I don't think that this is something that should have a separate PR.